OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -178,6 +178,7 @@ int main()
     {
         // Build the hopper.
         auto hopper = buildHopper();
+        hopper.finalizeFromProperties(); //required due to known bug
```

Comment:
I think this would be more accurate:
```cpp
        // Get a copy of the hopper built programmatically.
        auto hopper = buildHopper();
        // Finalize the hopper's definition, which includes identifying 
        // its subcomponents from its properties.
        hopper.finalizeFromProperties();
```